internals/overlord/planstate/manager.go

```diff
@@ -33,23 +32,19 @@ func (e *LabelExists) Error() string {
 }
 
 type PlanManager struct {
-	state     *state.State
-	runner    *state.TaskRunner
```

Comment:
Drive-by: PlanManager never used state and runner (maybe a copy 'n' paste issue), so remove them.